src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14;
+    private final int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        setColor(color);
+        obtainTheArea();
+    }
```

Comment:
Don't call methods like 'obtainTheArea' inside the constructor to set the area. It's better to set such properties directly in the method where they are needed or in a separate initialization block.